TELEMETRY/mAhRmn/mahRe2.lua

```diff
@@ -269,6 +274,7 @@ end
 local function init_func()
 	-- Called once when model is loaded
 	BatCapFullmAh = model.getGlobalVariable(GVBatCap, GVFlightMode) * 100
+	CapacityReservePercent = model.getGlobalVariable(GVCapResPer, GVFlightMode) --reads the reserved capacity from gv (sphrases)

```

Comment:
Could you add test
If ReadGVCapResPer then
CapacityReservePercent = model.getGlobalVariable(GVCapResPer, GVFlightMode
end

This way I hope to have fewer support questions :)